incubator/zalenium/README.md

```diff
@@ -0,0 +1,92 @@
+# ZALENIUM-HELM-CHART
+======================================================
+
+## Introduction
```

Comment:
we should have more instruction, how to run, how to get first impression on this chart,

https://opensource.zalando.com/zalenium/#usage